Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -260,231 +260,234 @@ bool CEXIMemoryCard::IsPresent() const
+        m_status &= ~MC_STATUS_READY;
 
         //???
 
         CmdDoneLater(5000);
       }
       break;
 
-    case cmdChipErase:
-      if (m_uPosition > 2)
+    case Command::ChipErase:
+      if (m_position > 2)
       {
         // TODO: Investigate on HW, I (LPFaint99) believe that this only
         // erases the system area (Blocks 0-4)
-        memorycard->ClearAll();
-        status &= ~MC_STATUS_BUSY;
+        m_memory_card->ClearAll();
+        m_status &= ~MC_STATUS_BUSY;
       }
       break;
 
-    case cmdPageProgram:
-      if (m_uPosition >= 5)
+    case Command::PageProgram:
+      if (m_position >= 5)
       {
-        int count = m_uPosition - 5;
+        int count = m_position - 5;
         int i = 0;
-        status &= ~MC_STATUS_BUSY;
+        m_status &= ~MC_STATUS_BUSY;
 
         while (count--)
         {
-          memorycard->Write(address, 1, &(programming_buffer[i++]));
+          m_memory_card->Write(m_address, 1, &(m_programming_buffer[i++]));
           i &= 127;
-          address = (address & ~0x1FF) | ((address + 1) & 0x1FF);
+          m_address = (m_address & ~0x1FF) | ((m_address + 1) & 0x1FF);
         }
 
         CmdDoneLater(5000);
       }
       break;
+
+    default:
+      break;
     }
   }
 }
 
 bool CEXIMemoryCard::IsInterruptSet()
 {
-  if (interruptSwitch)
-    return m_bInterruptSet;
+  if (m_interrupt_switch)
+    return m_interrupt_set;
   return false;
 }
 
 void CEXIMemoryCard::TransferByte(u8& byte)
 {
   DEBUG_LOG_FMT(EXPANSIONINTERFACE, "EXI MEMCARD: > {:02x}", byte);
-  if (m_uPosition == 0)
+  if (m_position == 0)
   {
-    command = byte;  // first byte is command
-    byte = 0xFF;     // would be tristate, but we don't care.
+    m_command = static_cast<Command>(byte);  // first byte is command
+    byte = 0xFF;                             // would be tristate, but we don't care.
 
-    switch (command)  // This seems silly, do we really need it?
+    switch (m_command)  // This seems silly, do we really need it?
     {
-    case cmdNintendoID:
-    case cmdReadArray:
-    case cmdArrayToBuffer:
-    case cmdSetInterrupt:
-    case cmdWriteBuffer:
-    case cmdReadStatus:
-    case cmdReadID:
-    case cmdReadErrorBuffer:
-    case cmdWakeUp:
-    case cmdSleep:
-    case cmdClearStatus:
-    case cmdSectorErase:
-    case cmdPageProgram:
-    case cmdExtraByteProgram:
-    case cmdChipErase:
+    case Command::NintendoID:
+    case Command::ReadArray:
+    case Command::ArrayToBuffer:
+    case Command::SetInterrupt:
+    case Command::WriteBuffer:
+    case Command::ReadStatus:
+    case Command::ReadID:
+    case Command::ReadErrorBuffer:
+    case Command::WakeUp:
+    case Command::Sleep:
+    case Command::ClearStatus:
+    case Command::SectorErase:
+    case Command::PageProgram:
+    case Command::ExtraByteProgram:
+    case Command::ChipErase:
       DEBUG_LOG_FMT(EXPANSIONINTERFACE, "EXI MEMCARD: command {:02x} at position 0. seems normal.",
```

Comment:
Note: recent versions of {fmt} format enum classes as integers so a cast is not required.